modules/reactor/include/meshgenerators/PeripheralRingMeshGenerator.h

```diff
@@ -1,85 +1,85 @@
-//* This file is part of the MOOSE framework
-//* https://www.mooseframework.org

```

Comment:
What is happening with GitHub's UI here?  It's telling me that this whole file is a diff, but everything under the giant swaths of red and green looks identical to me.

Reply:
Could be line-ending changes, or some other formatting change? There shouldn't be any content changes in PRMG from my work, I think this was introduced doing the clang-format at the end.

Reply:
Plain `git diff` on the branch, or even `git log -p`, gives me the same.

Reply:
Plain `diff` agrees ...
And yeah, it's format changes.  The original file was using dos `\n`, this one is using Unix `\n`.

So nothing to change on your side, I just need to use `git diff aaronoaks/PTMG-XY~~~..aaronoaks/PTMG-XY --ignore-cr-at-eol` to be able to actually see what's been modified.